unload_databricks_data_to_s3.py

```diff
@@ -7,8 +7,6 @@
 from pyspark.sql import DataFrame
 from pyspark.sql.functions import col
 
-MAX_RECORDS_PER_OUTPUT_FILE: int = 1_000_000
```

Comment:
line 106 still uses this constant. the corresopnding argument may still be used by java code